sdk/src/main/java/com/mapbox/maps/renderer/egl/EGLCore.kt

```diff
@@ -90,6 +90,14 @@ internal class EGLCore(
    * still current in a context.
    */
   fun releaseSurface(eglSurface: EGLSurface) {
+    if (egl.eglGetCurrentSurface(EGL10.EGL_DRAW) == eglSurface) {
```

Comment:
IMHO a bit too advance check, I'd simply put it inside 👇 `if` block.

Reply:
hlep